LayoutTests/fast/forms/switch/pointer-tracking-there-and-back-again.html

```diff
@@ -12,20 +13,22 @@
     if (UIHelper.isIOSFamily()) {
         const eventStreamData = new UIHelper.EventStreamBuilder()
             .begin(x, y)
-            .move(x + width, y, 0.1)
-            .move(x, y, 0.1)
+            .wait(0.21)
+            .move(x + width, y, 0.01)
+            .move(x, y, 0.01)
             .end()
             .takeResult();
         UIHelper.sendEventStream(eventStreamData);
         return;
     }
-    window.eventSender.mouseMoveTo(x, y);
-    window.eventSender.mouseDown();
-    window.eventSender.mouseMoveTo(x + width, y);
-    window.eventSender.mouseMoveTo(x, y);
-    window.eventSender.mouseUp();
+    await window.eventSender.mouseMoveTo(x, y);
+    await window.eventSender.mouseDown();
+    await window.eventSender.mouseMoveTo(x + width, y);
+    await window.eventSender.mouseMoveTo(x, y);
+    await window.eventSender.mouseUp();
```

Comment:
The synchronous version of these eventSender functions isn't awaitable.
```suggestion
    await window.eventSender.asyncMouseMoveTo(x, y);
    await window.eventSender.asyncMouseDown();
    await window.eventSender.asyncMouseMoveTo(x + width, y);
    await window.eventSender.asyncMouseMoveTo(x, y);
    await window.eventSender.asyncMouseUp();
```

Reply:
Do we even need to use the async versions?

Reply:
If the test passes with them there's no reason not to. I've been trying to adopt them in tests so that they behave correctly with the `--site-isolation` option.